Task - 1/Excercise2.java

```diff
@@ -0,0 +1,23 @@
+class A {
+	public A() {
+		System.out.println("Class A");
+	}
+}
+
+class B {
+	public B() {
+		System.out.println("Class B");
+	}
+}
+
```

Comment:
Can you add your observations about this exercise as a comment at the bottom of the program?